Assets/Scripts/FPSController.cs

```diff
@@ -19,11 +19,14 @@ public class FPSController : MonoBehaviour
     private Rigidbody rb;
     [SerializeField]
     private float jumpHeight = 2f;
+    [SerializeField]
```

Comment:
you have white space between other serialize fields, add here or remove elsewhere